### PR TITLE
Init Ledger RPC

### PR DESCRIPTION
**Pull request by otaliptus**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
   * [Configure Hardhat](developer-documentation/deployment-guide/configure-hardhat.md)
 * [System Contracts](developer-documentation/system-contracts/README.md)
   * [Bitcoin Light Client](developer-documentation/system-contracts/bitcoin-light-client.md)
+* [Rpc Documentation](developer-documentation/rpc-documentation/ledger-rpc-documentation.md)
```
Comment (orkunkilic) [inline SUMMARY.md:17]: Rpc -> RPC in everywhere

File: developer-documentation/rpc-documentation/ledger-rpc-documentation.md
Comment (orkunkilic) [inline developer-documentation/rpc-documentation/ledger-rpc-documentation.md:1]: Isn't there any good schemes to list this routes. I blieve there should be some markdown to create swagger like UI. This is ok, but can look better. Not urgent.

Comment (orkunkilic) [reply developer-documentation/rpc-documentation/ledger-rpc-documentation.md:1]: https://gist.github.com/azagniotov/a4b16faf0febd12efbc6c3d7370383a6

Comment (otaliptus) [reply developer-documentation/rpc-documentation/ledger-rpc-documentation.md:1]: Will do later.

**Pull request merged**